examples/README.md

```diff
@@ -0,0 +1,113 @@
+# Examples
+
+this directory contains examples for a few features of the autograder
+
+## Assessments
+
+The Hello folder and Hello.tar contain various examples of different styles of Assessments
+
+
+## Importing Assessments
+
+whether you are importing one or more assessments the procedure is the same
+
+multiple assessment import starter.tar is the starter file for importing one or more assessments
+
+
+```bash
+multiple assessment import starter.tar
+├── assessment1
+│   ├── assessment1.yml
+│   ├── autograde.tar
+│   └── autograde-Makefile
+└── assessment2
+    └── assessment1.yml
+
+```
+
+every folder in the tar represents a separate assessment
+
+assessment1 is an example of an assessment with an autograding component
+
+properties.yml
+```YAML
+---
+general:
+  name: assessment1
```

Comment:
Does this still have to match the directory name, or is the directory name now arbitrary?

Reply:
directory name is now arbitrary in the tar file